RX/DIY C3 Super Mini 2400.json

```diff
@@ -0,0 +1,27 @@
+{
+    "/serial_rx": 20,
```

Comment:
Got a few "/" sprinkled through this json.

Reply:
Yes as information so people know what the pins can be used for